Assets/Engine/Inventory/UI/ContainerUi.cs

```diff
@@ -46,6 +47,19 @@ private void UpdateContainer(Container container)
             float height = rectTransform.offsetMin.y + Math.Abs(rectTransform.offsetMax.y) + gridDimensions.y;
             var rect = transform.GetChild(0).GetComponent<RectTransform>();
             rect.sizeDelta = new Vector2(width, height);
+
+            
+
+            // Set the text inside the containerUI to be the name of the container
+            GameObject containerName = GameObject.Find("Container name");
+            Text containerNameText = containerName.GetComponent<Text>();
```

Comment:
Using Find() is bad practice because it is an expensive operation. Instead, the Text could be a public variable in the ContainerDisplay prefab.

In this case, we've got a more insidious issue. The code works fine when there is only one active object called "Container Name". However, when there is multiple (i.e. you've opened two containers), it can pick the wrong one! This causes the issue I wrote about in our Discord discussion. Again, making a variable for Text in the prefab instead of using Find will fix this.